packages/backend/src/config/parseConfig.ts

```diff
@@ -641,6 +642,9 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
             labels: getObjectFromEnvironmentVariable(
                 'LIGHTDASH_PROMETHEUS_LABELS',
             ),
+            queueMonitoringFrequency: getIntegerFromEnvironmentVariable(
+                'LIGHTDASH_QUEUE_MONITORING_FREQUENCY',
```

Comment:
Should we name it `LIGHTDASH_PROMETHEUS_QUEUE_MONITORING_FREQUENCY` and add it to the environment_variables.mdx ?